sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Generated/Models/ConnectedRegistryStatusDetail.cs

```diff
@@ -0,0 +1,86 @@
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="ConnectedRegistryStatusDetail"/>. </summary>
+        internal ConnectedRegistryStatusDetail()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="ConnectedRegistryStatusDetail"/>. </summary>
+        /// <param name="statusDetailType"> The component of the connected registry corresponding to the status. </param>
+        /// <param name="code"> The code of the status. </param>
+        /// <param name="description"> The description of the status. </param>
+        /// <param name="timestamp"> The timestamp of the status. </param>
+        /// <param name="correlationId"> The correlation ID of the status. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal ConnectedRegistryStatusDetail(string statusDetailType, string code, string description, DateTimeOffset? timestamp, string correlationId, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            StatusDetailType = statusDetailType;
+            Code = code;
+            Description = description;
+            Timestamp = timestamp;
+            CorrelationId = correlationId;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The component of the connected registry corresponding to the status. </summary>
+        [WirePath("type")]
+        public string StatusDetailType { get; }
+        /// <summary> The code of the status. </summary>
+        [WirePath("code")]
+        public string Code { get; }
+        /// <summary> The description of the status. </summary>
+        [WirePath("description")]
+        public string Description { get; }
+        /// <summary> The timestamp of the status. </summary>
+        [WirePath("timestamp")]
+        public DateTimeOffset? Timestamp { get; }
+        /// <summary> The correlation ID of the status. </summary>
+        [WirePath("correlationId")]
+        public string CorrelationId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).